app/types/recommendation.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * A crash recommendation object for the fatality review board 
+ * recommendation component
+ */
+export type Recommendation = {
+  id: number;
+  created_at: string;
+  created_by: string;
+  crash_pk: number;
+  rec_text: string | null;
+  rec_update: string | null;
+  recommendation_status_id: number | null;
+  atd__recommendation_status_lkp: RecommendationStatus | null;
+  recommendations_partners: RecommendationPartner[] | null;
+};
```

Comment:
We have some database tech debt that would be nice to fix sooner than later:

* Move `atd__recommendation_status_lkp` into the `lookups` schema
* Add `updated_at` and `updated_by` fields, and display them in the Recommendations card in the VZE UI